apps/condo/domains/acquiring/schema/MultiPayment.test.js

```diff
@@ -893,18 +894,35 @@ describe('MultiPayment', () => {
                 expect(multiPayment).toHaveProperty('status', MULTIPAYMENT_DONE_STATUS)
             })
         })
+        test('mobile resident can see his public data in his own MultiPayments', async () => {
+            const { admin, payments, acquiringIntegration, client } = await makePayerAndPayments()
+            const [createdMultiPayment] = await createTestMultiPayment(admin, payments, client.user, acquiringIntegration)
+            const multiPayment = await MultiPaymentResident.getOne(client, { id: createdMultiPayment.id })
+            expect(multiPayment).toBeDefined()
+            expect(multiPayment.id).toBe(createdMultiPayment.id)
+            expect(multiPayment.implicitFee).toBeUndefined()
+            expect(multiPayment.transactionId).toBeUndefined()
+            expect(multiPayment.meta).toBeUndefined()
```

Comment:
For better test coverage It is preferable to check all fields defined inside MultiPaymentResident

Reply:
if the business logic changes or another developer wants to change the access rights, then this test will be able to signal to him that these are the fields necessary for the resident